### PR TITLE
Magic string in adminEmail.js #4951

### LINKED ISSUE #4951: Magic string in adminEmail.js
In line 222,
See if we can remove "My alt text"
```
callbackFunction(url, {alt: 'My alt text'});
```

### PR DESCRIPTION
**Pull request by YongJieYongJie**
Fixes #4951

### CODE REVIEW

**Comment by YongJieYongJie** [comment]
This is where the placeholder is shown (I've modified it to be more informative):

image

File: src/main/webapp/js/common.js [hunk 1]
```diff
@@ -143,6 +143,10 @@ var DISPLAY_FEEDBACK_QUESTION_NUMSCALE_INTERVALINVALID = 'Please enter valid opt
 var DISPLAY_FEEDBACK_SESSION_VISIBLE_DATEINVALID = 'Feedback session visible date must not be empty';
 var DISPLAY_FEEDBACK_SESSION_PUBLISH_DATEINVALID = 'Feedback session publish date must not be empty';
 
+// Form input placeholders
+// Used in adminEmail.js only
+var PLACEHOLDER_IMAGE_UPLOAD_ALT_TEXT = 'Please enter an alt text for the image'
```
Comment (bamboo3250) [inline src/main/webapp/js/common.js]: I think we should put it in `adminEmail.js`.

File: src/main/webapp/js/common.js [hunk 2]
```diff
@@ -143,6 +143,10 @@ var DISPLAY_FEEDBACK_QUESTION_NUMSCALE_INTERVALINVALID = 'Please enter valid opt
 var DISPLAY_FEEDBACK_SESSION_VISIBLE_DATEINVALID = 'Feedback session visible date must not be empty';
 var DISPLAY_FEEDBACK_SESSION_PUBLISH_DATEINVALID = 'Feedback session publish date must not be empty';
 
+// Form input placeholders
+// Used in adminEmail.js only
```
Comment (bamboo3250) [inline src/main/webapp/js/common.js]: No need to mention where it is used.

**Comment by YongJieYongJie** [comment]
@bamboo3250 Updated and ready for review!

File: src/main/webapp/js/adminEmail.js [hunk 3]
```diff
@@ -1,5 +1,7 @@
-var callbackFunction;
+// Form input placeholders
+var PLACEHOLDER_IMAGE_UPLOAD_ALT_TEXT = 'Please enter an alt text for the image'
```
Comment (bamboo3250) [inline src/main/webapp/js/adminEmail.js]: A semi-colon at the end of line?

**Comment by YongJieYongJie** [comment]
@bamboo3250 
Thanks for spotting that! (too much ruby for me)

File: src/main/webapp/js/adminEmail.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
-var callbackFunction;
+// Form input placeholders
+var PLACEHOLDER_IMAGE_UPLOAD_ALT_TEXT = 'Please enter an alt text for the image';
```
Comment (bamboo3250) [inline src/main/webapp/js/adminEmail.js]: Is there any reason for changing the text?

Comment (YongJieYongJie) [inline src/main/webapp/js/adminEmail.js]: I thought "My alt text" is not very informative (see the screenshot above).
A call-to-action may prompt the admin to actually enter a proper alt text.

**Comment by bamboo3250** [comment]
I see. Good to merge :+1:

**Comment by thyageshm** [comment]
@bamboo3250 ready for merge :-)

**Pull request merged**